specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -24,158 +25,105 @@
 
 ## Summary
 
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
-
-At the moment customers can deploy and manage jobs through the VDK CLI tool or through the API's.
-We want to extend on this and create a web frontend for the VDK.
-The frontend will let customers, create,delete deploy and generally manage and browse their jobs.
-Frontend is written in an extensible format to easily allow other teams to build extensions.
-To achieve this extensibility the frontend is built in angular.
+At the moment customers can deploy and manage jobs through the VDK CLI tool or
+the Control Service API. Adding a web UI to the VDK lets customers develop and
+manage data jobs more comfortably. The Operations UI packages are built-in
+Angular and are designed to be extensible. This allows anyone to build their
+project on top of the existing UI components.
 
 ## Glossary
-<!--
-Optional section which defines terms and abbreviations used in the rest of the document.
--->
 
-data-pipelines: This is the npm project of the bootstrapping codebase it contains all the vdk specific frontend code. for more details please see [README.md](/projects/frontend/data-pipelines/README.md)
-shared-components, @versatiledatakit/shared: This is a set of components that are used within the data-pipelines project but are generic and could be re-used by other projects [README.md](/projects/frontend/data-pipelines/README.md)
+@versatiledatakit/data-pipelines: npm project for bootstrapping of the codebase.
```

Comment:
I don't understand why changing `data-pipelines` project dir name, to the artifact published name? 

Also, appreciating how clever it is to have an empty PR description (since documentation changes only, so they are self-explanatory) 😊. However, here I see "style and grammar" change, and I see other multiple types of changes, and that seems like a discrepancy. May you add to PR description what types of changes were made?

Reply:
Fair point, I'll add it to the description.